backend/routes/login.js

```diff
@@ -0,0 +1,22 @@
+const Router = require('express');
+const { checkSchema } = require('express-validator');
+const { validateLogin } = require('../handlers/login');
+const { handleInputErrors } = require('../modules/middlewares');
+
+const loginRouter = Router();
+
+loginRouter.post(
+  '/login',
+  checkSchema({
+    username: {
+      isString: true,
+    },
+    password: {
+      isString: true,
+    },
+  }),
```

Comment:
non blocking but these schemas can also be defined as separate variables - useful when the schemas are really large and complex and it can be nice to have them defined elsewhere to not cluter up the API route handler itself

Reply:
Created a variable called loginSchema in the file and assigned it the object of the different validators(username and password isString) then passed loginSchema into the checkSchema function, reducing clutter in the api route.